lib/os_io.c

```diff
@@ -37,6 +37,199 @@ ___io_module ___io_mod =
 
 /*---------------------------------------------------------------------------*/
 
+/* poll dynamic fdset memory management. */
+
+#ifdef USE_select_or_poll
+
+___HIDDEN int ___fdset_heap_overflow;
```

Comment:
Actually, there are good reasons for the size being in vmos -- we can't end up with some processors using a different fdset than others, because of some allocation failure in one of them.